man/man5/zfs-module-parameters.5

```diff
@@ -1752,14 +1769,97 @@ Default value: \fB2\fR.
 .ad
 .RS 12n
 Max concurrent I/Os per top-level vdev (mirrors or raidz arrays) allowed during
-scrub or resilver operations.
+legacy scrub or resilver operations.
 .sp
 Default value: \fB32\fR.
 .RE
 
 .sp
 .ne 2
 .na
+\fBzfs_scan_md_maxinflight\fR (int)
+.ad
+.RS 12n
+Max concurrent I/Os allowed during scrub or resilver metadata scanning phase.
+.sp
+Default value: \fB1024\fR.
```

Comment:
I think it makes more sense for the limit to be expressed as "per disk" or "per top-level vdev" (which are essentially the same for scrub i/os, at least), because performance (approximately) scales with # vdevs.  Rather than being expressed "per pool".

I'm fine with changing the name to make it more clear.  Or if you have a proposal for making it actually limit this number on each vdev, rather than just limiting the average number per vdev, I'm OK with that too.